File: Java/app/src/main/java/com/example/qispace_sdk_java_demo/MasqActivity.java
package com.example.qispace_sdk_java_demo;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.annotation.Nullable;

public class MasqActivity extends Activity {
    @Override
    public void onCreate(@Nullable Bundle savedInstanceState){
        super.onCreate(savedInstanceState);
        setContentView(R.layout.masq_layout);

        findViewById(R.id.button_masq_back).setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View view) {
                Intent backToLanding = new Intent(MasqActivity.this, MainActivity.class);
                startActivity(backToLanding);
            }
        });
    }
}

File: c/Kotlin/app/src/main/kotlin/com/example/demoqispaceapp/masqActivity.kt
package com.example.demoqispaceapp

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import android.widget.Button

class masqActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?){
        super.onCreate(savedInstanceState)
        setContentView(R.layout.masq_layout)

        // BUTTON: Send user back to landing page
        findViewById<Button>(R.id.button_masq_back).setOnClickListener{
            val toLanding = Intent(this, MainActivity::class.java)
            startActivity(toLanding)
        }
    }

}